### PR TITLE
fixed expired certificate

### PR DESCRIPTION
**Pull request by AndysWorth**
Let’s Encrypt SSL certificate expired and is causing a "bad handshake" error.  This updates the certificate.

Tests were also updated so that all tests pass.

### CODE REVIEW

File: tests/platform_tests/bids-apps_BIDS_multi_session_session_ses-Session2.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import flywheel
+
+
+input_files = {}
+
+
+def main(fw):
+
+    gear = fw.lookup("gears/bids-freesurfer")
+    print("gear.gear.version for job was = 1.0.4_6.0.1-5")
```
Comment (neurosutton) [inline tests/platform_tests/bids-apps_BIDS_multi_session_session_ses-Session2.py:23]: "gear.gear.version for originally generated test job was" ?

File: tests/integration_tests/test_run.py [hunk 2]
```diff
@@ -122,6 +122,11 @@ def test_dry_run_works(caplog):
     user_json = Path(Path.home() / ".config/flywheel/user.json")
     if not user_json.exists():
         TestCase.skipTest("", f"No API key available in {str(user_json)}")
+    with open(user_json) as json_file:
+        data = json.load(json_file)
+        if "ga" not in data["key"]:
+            TestCase.skipTest("", "Not logged in to ga.")
+
```
Comment (neurosutton) [inline tests/integration_tests/test_run.py:129]: Nice!

**Review by neurosutton — APPROVED** [review]
LGTM

File: tests/platform_tests/bids-apps_BIDS_multi_session_session_ses-Session2.py (hunk 1 above)
Comment (AndysWorth) [reply tests/platform_tests/bids-apps_BIDS_multi_session_session_ses-Session2.py:23]: Good idea!

**Pull request merged**